client/src/featureform/serving.py

```diff
@@ -212,6 +219,13 @@ def _get_source_columns(self, name, variant):
         resp = self._stub.SourceColumns(req)
         return resp.columns
 
+    def _nearest(self, name, variant, vector, k):
```

Comment:
so this supports nearest by vector. Just out of curiosity, how hard would it be to do nearest by entity ID?

Reply:
I don't want to promise anything given the time remaining before the release, but I honestly don't think it would be too hard to implement.